Resources/Prototypes/_NF/Entities/Structures/Machines/Computers/computers_shipyard_mothership.yml

```
@@ -15,9 +15,10 @@
   - type: ShipyardListing
 
 # The empress console
+# Does NOT extend BaseMothershipComputer because the NFSD requires special stuff.
 - type: entity
   id: EmpressMothershipComputer
-  parent: BaseMothershipComputer
+  parent: ComputerShipyardNfsd
```

Regarding the comment above, wouldn't multiple inheritance fix this?

Need to double check w.r.t. inheritance ordering (assuming order runs left-to-right)
```suggestion
  parent: [BaseMothershipComputer, ComputerShipyardNfsd]
```

Might want to add a separate suffix and description here that's empress-specific.

---

That does not seem to work correctly, unfortunately. But you did highlight an error in my implementation: description and Mothership suffix are not present with the current version. I'll see if I can fix this in... some way, without introducing a bajillion new prototypes.

---

I pushed an update that extends `BaseMothershipComputer` only, and duplicates a little bit of stuff from the NFSD shipyard console. It was the simplest and most reliable way to get functionality from both, without introducing a load of annoying extra stuff.